.github/workflows/builds.yml

```
@@ -38,9 +38,14 @@ on:
         type: boolean
         description: 'Observer'
         required: false
-      include_without_netgo:
+      include_with_adx:
+        type: boolean
+        description: 'Build `with_adx` images'
+        required: false
+      # GHA allows only up to 10 inputs - regroup two entries in one
+      include_without_netgo_without_adx:
```

how about renaming this to
```suggestion
      include_alternative_builds:
```

then when set, it build all alternative builds (`without_adx`, `without_netgo_without_adx`, `arm`)

---

implemented in https://github.com/onflow/flow-go/pull/5254/commits/f1e015e30d3ca9975c0bac755dc5f46e9263b3e8 and https://github.com/onflow/flow-go/pull/5254/commits/a3951090c1f53ea324f229d04f623c80ff42ea95